src/reducers/order.js

```diff
@@ -13,20 +13,13 @@ export default function(state = initialState, action) {
       return state.filter((product, index) => {
         return index !== action.payload.index
       })
-    case INCREASE_PRODUCT_QUANTITY:
+    case UPDATE_PRODUCT_QUANTITY:
       return state.map((product, index) => {
         return index === action.payload.index ? { 
           ...product,
-          quantity: product.quantity + 1 <= 10 ? product.quantity + 1 : 10
+          quantity: action.payload.quantity
          } : product
-      })
```

Comment:
Forgot to add it in my previous review, but we can refactor this a little. Consider applying this approach:
```javascript
const orders = [...]

const newOrders = [...orders]
newOrders[index].quantity = action.payload.quantity

return newOrders
```